addons/armaos/functions/fnc_os_clear.sqf

```diff
@@ -1,32 +1,17 @@
-params ["_options", "_consoleInput"];
+params ["_computer", "_options"];
 
-_result = [];
-
-_optionsCount = count _options;
+private _result = [];
 
-scopeName "main";
+if (count _options >= 1) exitWith {["   Command: clear has no options"];};
 
-switch (true) do
-{
-	case (_optionsCount >= 1):
-	{
-		//hint "Case 1";
-		
-		_result = ["   Command: clear has no options"];
-		_result breakOut "main";
-	};
-	case (_optionsCount == 0):
-	{
-		//hint "Case 2";
+private _terminal = _computer getVariable "AE3_terminal";
 
-		_outputText = [] call AE3_fnc_headerText;
-		_pointer = _consoleInput getVariable ["AE3_filepointer", []];
-		_outputText = _outputText + (_pointer joinString "/") + "> ";
+_terminal set ["AE3_terminalBuffer", []];
+_terminal set ["AE3_terminalCursorLine", 0];
+_terminal set ["AE3_terminalCursorPosition", 0];
 
-		ctrlSetText [1100, _outputText];
+_computer setVariable ["AE3_terminal", _terminal];
```

Comment:
Should be unnecessary, because variables referencing hashmaps are only pointers to the actual hashmap object, not the hashmap itself.

Reply:
removed in https://github.com/y0014984/Advanced-Equipment/pull/90/commits/07962c1e890a951964741bf3062f8574b389741c